src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -198,6 +201,21 @@ private async Task<string> UploadArchiveToAws(string bucketName, string archiveP
         return archiveBlobUrl;
     }
 
+    private async Task<string> UploadArchiveToGithub(string org, string archivePath)
+    {
+
+        await using (var archiveData = _fileSystemProvider.OpenRead(archivePath))
+        {
+            var isMultipart = archiveData.Length > STEAM_SIZE_LIMIT; // Determines if stream size is greater than 100MB
+
+            _log.LogInformation($"Uploading archive to GitHub Storage");
+            var keyName = GenerateArchiveName();
+            var authenticatedGitArchiveUri = await _githubApi.UploadArchiveToGithubStorage(org, isMultipart, keyName, archiveData);
```

Comment:
Don't have a strong opinion here but wouldn't it be better to delegate the `isMultipart` to the `UploadArchiveToGithubStorage` method? that way we won't need to pass the boolean flag and the method itself will decide whether to use multipart content or not.

Reply:
I agree! [2894bb8](https://github.com/github/gh-gei/pull/1280/commits/2894bb8c235f37f66a573e36bec50952750deeec)